z3c/dependencychecker/modules.py

```diff
@@ -115,11 +116,31 @@ def _process_ast_node(self, node):
                     file_path=self.path,
                     is_test=self.testing,
                 )
+        elif isinstance(node, ast.Try):
+            if not HAS_REPORTED_CONDITIONAL_IMPORTS:
+                self._is_a_conditional_import(node)
 
     @staticmethod
     def _is_relative_import(import_node):
         return import_node.level > 0
 
+    def _is_a_conditional_import(self, node):
+        global HAS_REPORTED_CONDITIONAL_IMPORTS
```

Comment:
Normally `CONSTANTS` are pretty constant and don't change. This one does, which surprised me initially.
A short comment next to the original constant would help (`# Gets set to True when xyz is found`).